images/router/haproxy/conf/haproxy-config.template

```diff
@@ -230,7 +230,7 @@ backend be_edge_http_{{$cfgIdx}}
   {{ end }}
   http-request set-header Forwarded for=%[src];host=%[req.hdr(host)];proto=%[req.hdr(X-Forwarded-Proto)]
                 {{ range $idx, $endpoint := endpointsForAlias $cfg $serviceUnit }}
-  server {{$endpoint.IdHash}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter 5000ms cookie {{$endpoint.IdHash}}
+  server {{$endpoint.IdHash}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter {{env "ROUTER_BACKEN_CHECK_INTER" "5000ms"}} cookie {{$endpoint.IdHash}}
```

Comment:
And should it be HAPROXY_ROUTER_BACKEND_CHECK_INTERVAL to be consistent with the SYSLOG variables?